pkg/cluster/spec.go

```diff
@@ -277,19 +277,14 @@ func NewSpecFromClusterConfig(clusterConfigPath string, cliVersion version.Info,
 	return s, nil
 }
 
-func BuildSpecFromBundles(cluster *eksav1alpha1.Cluster, bundles *v1alpha1.Bundles, opts ...SpecOpt) (*Spec, error) {
+func BuildSpecFromBundles(cluster *eksav1alpha1.Cluster, bundles *v1alpha1.Bundles, eksd *eksdv1alpha1.Release, opts ...SpecOpt) (*Spec, error) {
```

Comment:
Can we pass the eksd release in an option?

And then have a nil check and if the eksd release is not there, then retrieve it